.github/workflows/eerelease.yml

```diff
@@ -0,0 +1,127 @@
+name: Release Ansible Execution Environment
+on:
+  workflow_dispatch:
+    inputs:
+      ee-name:
+        description: 'ee name'
+        required: true
+        type: choice
+        options:
+          - community-ee-base
+          - community-ee-minimal
+      ansible-core-version:
+        description: >-
+          Release Version. Example: 11.1.0
+        required: true
+      ee-version:
+        description: >-
+          EE Release Version. Example: 1
+        required: true
+      tag_as_latest:
+        description: >-
+          Whether to tag the image as latest.
+        type: boolean
+        required: false
+        default: false
+
+env:
+  EE_VERSION: ${{ inputs.ansible-core-version }}-${{ inputs.ee-version }}
+
+jobs:
+  build:
+    name: Build Execution Environment ${{ inputs.ansible-core-version }}-${{ inputs.ee-version }}
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+      packages: write
+
+    steps:
+      - name: Check out images
+        uses: actions/checkout@v4
+        with:
+          repository: ansible-community/images
+          ref: main
+          path: image
+
+      - name: Check out eercheck repo for testing
+        uses: actions/checkout@v4
+        with:
+          repository: anweshadas/eercheck
```

Comment:
Since the code in that repository is specifically about checking the two images from this repo, it might make sense to move it to this repo. (But there's no need to do it now.)

Reply:
Okay, once this PR is merged, I will file another PR to move this repo here.